src/Microsoft.Azure.Relay/WebSockets/NetCore/WebSocketHandle.Managed.cs

```diff
@@ -80,16 +91,28 @@ public void Abort()
             // TODO #14480 : Not currently implemented, or explicitly ignored:
             // - ClientWebSocketOptions.UseDefaultCredentials
             // - ClientWebSocketOptions.Credentials
-            // - ClientWebSocketOptions.Proxy
             // - ClientWebSocketOptions._sendBufferSize
 
             // Establish connection to the server
             CancellationTokenRegistration registration = cancellationToken.Register(s => ((WebSocketHandle)s).Abort(), this);
             try
             {
-                // Connect to the remote server
-                Socket connectedSocket = await ConnectSocketAsync(uri.Host, uri.Port, cancellationToken).ConfigureAwait(false);
-                Stream stream = new AsyncEventArgsNetworkStream(connectedSocket);
+
+                Socket connectedSocket;
+                Stream stream;
+                if (options.Proxy != null && options.Proxy != WebRequest.DefaultWebProxy)
```

Comment:
If there's a proxy set via config(?) or IE settings why not try to use it?

Reply:
This requires interop native calls, and not top of our priority.